api/services/InvitationService.js

```diff
@@ -155,13 +155,14 @@ module.exports = {
     }
   },
 
-  use: (userId, token, accessCode) => {
+  async use (userId, token, accessCode) {
```

Comment:
this method used to throw a plain unwrapped error.  are you sure that it's calling function handles it correctly now with async?